src/Transformation/Factory.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/* Copyright (c) 2017 Richard Klees <richard.klees@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Transformation;
+
+/**
+ * Factory for basic transformations.
```

Comment:
Hm, I', not allowed to comment on [this one](https://github.com/ILIAS-eLearning/ILIAS/pull/476#discussion_r111383488). I agree, @shecken, would you please?